src/main.c

```diff
@@ -154,6 +154,8 @@ int main(void) {
     }
   }
 
+  core_init_app_registry();
```

Comment:
can we add this function call inside `application_init`?

Reply:
Not it literally makes sense

Reply:
updated